### PR TITLE
Update SemDedup docs

### PR DESCRIPTION
**Pull request by sarahyurick**
Closes https://github.com/NVIDIA/NeMo-Curator/issues/203

### CODE REVIEW

File: docs/user-guide/semdedup.rst [hunk 1]
```diff
@@ -118,32 +118,32 @@ The semantic deduplication process is controlled by two key threshold parameters
...
+1. ``eps_thresholds``: A list of similarity thresholds used to compute semantic matches. Each threshold represents a different level of strictness in determining duplicates.
                      Lower values are more strict, requiring higher similarity for documents to be considered duplicates.
 
-2. `eps_to_extract`: The specific threshold used for the final extraction of deduplicated data.
-                     This value must be one of the thresholds listed in `eps_thresholds`.
+2. ``eps_to_extract``: The specific threshold used for the final extraction of deduplicated data.
+                     This value must be one of the thresholds listed in ``eps_thresholds``.
 
 This two-step approach offers several advantages:
-* Flexibility to compute matches at multiple thresholds without rerunning the entire process.
```
Comment (ryantwolf) [inline docs/user-guide/semdedup.rst:128]: I think `*` is the right format for unordered lists in `rst` files. I think we just need a blank line before the list begins. See [here](https://github.com/NVIDIA/NeMo-Curator/blob/main/docs/user-guide/download.rst).

File: docs/user-guide/semdedup.rst [hunk 2]
```diff
@@ -118,32 +118,32 @@ The semantic deduplication process is controlled by two key threshold parameters
...
-* Ability to analyze the impact of different thresholds on your dataset.
-* Option to fine-tune the final threshold based on specific needs without recomputing all matches.
+- Flexibility to compute matches at multiple thresholds without rerunning the entire process.
+- Ability to analyze the impact of different thresholds on your dataset.
+- Option to fine-tune the final threshold based on specific needs without recomputing all matches.
 
 When choosing appropriate thresholds:
-* Lower thresholds (e.g., 0.001): More strict, resulting in less deduplication but higher confidence in the identified duplicates.
-* Higher thresholds (e.g., 0.1): Less strict, leading to more aggressive deduplication but potentially removing documents that are only somewhat similar.
+- Lower thresholds (e.g., 0.001): More strict, resulting in less deduplication but higher confidence in the identified duplicates.
```
Comment (ryantwolf) [inline docs/user-guide/semdedup.rst:133]: Same comment about unordered lists

**Review by ryantwolf — CHANGES_REQUESTED** [review]
Thanks for touching this up. There's one thing to change but other than that looks good.

File: docs/user-guide/semdedup.rst (hunk 1 above)
Comment (sarahyurick) [reply docs/user-guide/semdedup.rst:128]: Updated, thanks!

**Comment by sarahyurick** [comment]
Hi @ryantwolf this is ready for review again.

**Review by ryantwolf — APPROVED** [review]
Looks good, thanks!

**Pull request merged**